### PR TITLE
[Look&Feel] Consistency of Plus Icons

### PR DESCRIPTION
**Pull request by danieldong51**
### Description

Updated plus icon over plusInCircle for add/create use cases. Did not change instances of other use cases, or when minusInCircle was attached. 

## Screenshots

| Scope | Before (v7dark) | After (v7dark) | Before (v8dark) | After (v8dark) |
| ----- | ----- | ----- | ----- | ----- |
| Dashboards: New dashboard creation | Dashboards 1 v7 Before | Dashboards 1 v7 After | Dashboards 1 v8 Before | Dashboards 1 v8 After |
| Dashboards: Visualization creation | Dashboards 2 v7 Before | Dashboards 2 v7 After | Dashboards 2 v8 Before | Dashboards 2 v8 After |

## Changelog
- refactor: [Look&Feel] Consistency of Plus Icons

### Check List

- [x] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/dashboard/public/application/components/dashboard_listing/create_button.tsx [hunk 1]
```diff
@@ -48,8 +48,7 @@ const CreateButton = (props: CreateButtonProps) => {
   const renderCreateMenuDropDown = () => {
     const button = (
       <EuiButton
-        iconType="arrowDown"
```
Comment (virajsanghvi) [inline src/plugins/dashboard/public/application/components/dashboard_listing/create_button.tsx:51]: Can we not change arrowDown - this is a dropdown so remain the same

**Review by virajsanghvi — COMMENTED** [review]
Looks like a linter error?

Also, I started a thread with Laura to validate these two use cases (I could see us not wanting to change these):
image

**Review by zhongnansu — COMMENTED** [review]
Could you check the failed CI? I think with your change, the snapshot testing are failing and it needs update.
```
Snapshot Summary
 › 2 snapshots failed from 2 test suites. Inspect your code changes or run `yarn run test:jest:ci:coverage -u` to update them.
```

**Comment by virajsanghvi** [comment]
You likely want to rerun the failing tasks - looks like its failing on linux, but passing on windows.

Main blocker on my side to approving is Laura's feedback on the icons that are on list items. Will ping her again

**Comment by virajsanghvi** [comment]
I restarted failed tests to see if they'd pass - think this one needs the changelog entry as well

**Review by virajsanghvi — APPROVED** [review]

**Review by zhongnansu — APPROVED** [review]

**Pull request merged**